<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RizinPluginsDialog</class>
 <widget class="QDialog" name="RizinPluginsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rizin plugin information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="RzBinTab">
      <attribute name="title">
       <string>RzBin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="RzBinLabel">
         <property name="text">
          <string>RzBin plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RzBinTreeWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RzIOTab">
      <attribute name="title">
       <string>RzIO</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="RzIOLabel">
         <property name="text">
          <string>RzIO plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RzIOTreeWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Permissions</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RzCoreTab">
      <attribute name="title">
       <string>RzCore</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="RzCoreLabel">
         <property name="text">
          <string>RzCore plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RzCoreTreeWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RzAsmTab">
      <attribute name="title">
       <string>RzAsm</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="RzAsmLabel">
         <property name="text">
          <string>RzAsm plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RzAsmTreeWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Architecture</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CPU's</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RizinPluginsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
